docs/configuration/configuration.md

```diff
@@ -429,7 +429,28 @@ region: <string>
 # The port to scrape metrics from. If using the public IP address, this must
 # instead be specified in the relabeling rule.
 [ port: <int> | default = 80 ]
+
+# Filters can be used optionally to filter the instance list by other criteria.
+# Available filter criteria can be found here:
+# https://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_DescribeInstances.html
+# Filter API documentation: https://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_Filter.html
+# For example if we need to filter instances with tags environment=production and team=backend,
+# we can use the following filters:
+# filters:
+# - name: tag:environment
+#   values:
+#   - production
+# - name: tag:team
+#   values:
+#   - backend
+filters:
```

Comment:
I meant to expand this out, not to put an example in the comments.

Reply:
👍 But still not sure how to expand it out, could you please give me a hint?

Reply:
Maybe something like:

```
[ - name: <string>
    values: <string>, [...] ]
```

Reply:
@brian-brazil It's really unfortunate that I had to go to the pull request to see an example of the syntax on this. It's not clear from the currently existing documentation and no examples are provided.

I wish this config example had made it into the documentation it was very helpful.